### PR TITLE
C_CPP : Storage Classes in C++

### LINKED ISSUE #3988: C_CPP : Storage Classes in C++
### Description

Documentation on <b>storage classes in C++</b>.

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by vartikavr**
<hr>

## Description 📜

This documentation contains explanations along with examples of storage classes in C++.
Fixes #3988 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Fundamentals/Storage Classes in C++/Storage_Classes.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Storage Classes
+
+## What are storage classes?
+
+Storage classes are used to describe the different features like scope, visibility, and lifetime of a variable or a function in a C++ program. These features tell the existence and use of a variable (or a function) during the runtime of the program.</br>
+There are <strong>five types of storage classes</strong> that are used in C++, these are :
+
+## 1. Automatic
+
+The keyword used for this storage class is : <em>`auto`</em>. The lifetime that this storage class offers to a variable or function is : <strong>function block</strong>. </br>
```
Comment (apeksha235) [inline C_CPP/Fundamentals/Storage Classes in C++/Storage_Classes.md:10]: Get rid of the html tags here

File: C_CPP/Fundamentals/Storage Classes in C++/Storage_Classes.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+### Output
+
+```
+GWOC - 2021
+```
+
+## 2. External
+
+The keyword used for this storage class is : <em>`extern`</em>. The lifetime that this storage class offers to a variable or function is : <strong>the whole program</strong>. </br>
```
Comment (apeksha235) [inline C_CPP/Fundamentals/Storage Classes in C++/Storage_Classes.md:37]: Here as well

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@vartikavr Just make the following changes, excellent work otherwise!

**Comment by vartikavr** [comment]
@apeksha235 I've done the suggested changes. Please look into it.

**Review by deeqakkk — APPROVED** [review]
Nicely explained👏
@Saviour1001 its good to be merged.

**Pull request merged**